packages/fxa-settings/src/components/ExternalAlert/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { AlertBar } from '../AlertBar';
+import { useAccount } from '../../models';
+import { externalAlert } from '../../lib/cache';
+
+export const ExternalAlert = () => {
```

Comment:
We _could_ call this`AlertExternal` instead of `ExternalAlert` - that way in the component tree list, it's next to `AlertBar`. We've been trying to stick with `[noun][descriptor]` rather than `[descriptor][noun]` for that reason but not sure that'll be easy to keep up with conventionally so I'm fine with either. If we leave this `ExternalAlert` it temps me to change `LinkExternal` to `ExternalLink`.